test/jasmine/tests/click_test.js

```diff
@@ -47,7 +47,7 @@ describe('Test click interactions:', function() {
             setTimeout(function() {
                 click(x, y);
                 setTimeout(function() { resolve(); }, DBLCLICKDELAY / 2);
-            }, DBLCLICKDELAY / 4);
+            }, DBLCLICKDELAY / 2);
```

Comment:
Looks like you fixed https://github.com/plotly/plotly.js/issues/567 Thanks!

Maybe we could listen to [`plotly_afterplot`](https://github.com/plotly/plotly.js/blob/76298379da2bbdd96066a4a28fc633ff98fe7d9b/src/plot_api/plot_api.js#L311) ?